simulate-event.js

```diff
@@ -340,7 +339,7 @@ module.exports = function (element, type, options) {
   options = extend({
     bubbles: true,
     cancelable: true
-  }, eventOptions[eventType](element, type, options), options)
+  }, eventOptions[eventType](type, options), options)
```

Comment:
But won't some options not be set yet (like those that come from calling `eventOptions[eventType]`)? Just wondering if that matters?